ragatouille/models/index.py

```diff
@@ -201,13 +180,73 @@ def build(
 
         # Instruct colbert-ai to disable forking if nranks == 1
         self.config.avoid_fork_if_possible = True
-        indexer = Indexer(
-            checkpoint=checkpoint,
-            config=self.config,
-            verbose=verbose,
-        )
-        indexer.configure(avoid_fork_if_possible=True)
-        indexer.index(name=index_name, collection=collection, overwrite=overwrite)
+
+        # Monkey-patch colbert-ai to avoid using FAISS
+        monkey_patching = False
```

Comment:
can we refactor to `monkey_patching = len(collection) < 500000 and kwargs.get("use_faiss", False) is False`?

Then L86 becomes `if monkey_patch:`  and we can remove L203

Reply:
Yep, I'd be fine with that!

Reply:
Done